certify-mock-identity.properties

```diff
@@ -127,31 +127,6 @@ mosip.certify.key-values={\
                                   'text_color': '#FFFFFF'}},\
                     'order' : {'fullName','phone','dateOfBirth','gender','email','region','province','UIN', 'VID', 'postalCode'}\
                  },\
```

Comment:
please remove the comma in L129?
`}\`
Please add as above

Reply:
updated